packages/client/src/v2-events/features/events/useEventConfiguration.ts

```diff
@@ -18,7 +18,10 @@ import { useTRPC } from '@client/v2-events/trpc'
  */
 export function useEventConfigurations() {
   const trpc = useTRPC()
-  const config = useSuspenseQuery(trpc.event.config.get.queryOptions()).data
+  const config = useSuspenseQuery({
```

Comment:
A small fix so that the query keys are formed as

```
[["event","config","get"]]
```

and not as 

```
[["event","config","get"],{"type":"query"}]
```


```
[["event","config","get"],{"type":"query"}]
```

```
[["event","config","get"],{"type":"query"}]
```

Saw a few places where cached values were either accessed or manipulated that didn't take this into account = cached data was not found